### DISCUSSION TITLE
what happen why it show "None of your plots have passed the plot filter yet" in Farm Overview

### DISCUSSION THREAD

**Discussion by aircoolza** [Chia-Network/chia-blockchain#4766] [Support]
<h2>help me please </h2>

Farm overview in Last Attempted Proof I'm not active  please tell me how to fix 

![รูปที่1](https://www.img.in.th/images/120bf74c582c6329bba203f052ea0871.png)

**Comment by googgiig9908** [comment]
i do have the same Ploblem
and one more thing (Your full node connection )is not show up
we need help!!!! then I think we can farm but can't get the token

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
